client/signup/config/flows-pure.js

```diff
@@ -122,21 +123,6 @@ export function generateFlows( {
 			lastModified: '2019-04-30',
 		},
 
-		'onboarding-for-business': {
```

Comment:
I know this is work in progress, just flagging that we reference `onboarding-for-business` explicitly in several places:

https://github.com/Automattic/wp-calypso/blob/16546dde27c1eac2bd1eddf6cf49719b61027e91/client/lib/signup/page-builder.js#L38

https://github.com/Automattic/wp-calypso/blob/6b391cd07f5c323b51a10ea7337593709ac6cd66/client/signup/config/flows.js#L245

https://github.com/Automattic/wp-calypso/blob/039246b7dbf7ee4bf95c88fa30c55898a7653d8c/client/lib/signup/step-actions.js#L184

Probably pretty easy to remove or modify these blocks... :)